CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+7. Review notes from team and update PR accordingly
+8. Once PR has been approved, the branch is deleted.
+
+## Contributions and License
+
+By contributing, you agree that your contributions will be licensed under its MIT License.
+
+Any contributions you make will be under the MIT Software License.
+
+In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+## Reporting Bugs
+
+Report bugs using Github [issues](https://github.com/cds-amal/fzf-eip/issues).
+
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](https://github.com/cds-amal/fzf-eip/issues). It's super easy!
+
+### Bug reports format
+
+Write bug reports with detail, background, and sample code.
+
+[This is an example](http://stackoverflow.com/q/12488905/180626) of a bug report written by someone else. We also think it's not a bad model. Here's [another example from Craig Hockenberry](http://www.openradar.me/11905408), another app developer.
+
+## Bug Triage
+
+Here are some tags that we're using to better organize issues in this repo:
+
+- `good first issue` - Good candidates for someone new to the project to contribute.
+- `help wanted` - Issues that should be addressed and which we would welcome a
+  PR for but may need significant investigation or work
+- `support` - Request for help with a concept or piece of code but this isn't an
+  issue with the project.
+- `needs more info` - Missing repro steps or context for both project issues \&
+  support questions.
+- `discussion` - Issues where folks are discussing various approaches \& ideas.
+- `question` - Something that is a question specifically for the maintainers such
+  as [this issue about the license](https://github.com/facebook/draft-js/issues/1819).
+- `documentation` - Relating to improving documentation for the project.
+
+* Browser \& OS-specific tags for anything that is specific to a particular
+  environment (e.g. `macos`, `unbuntu`, `windows`, etc).
+
+### Summary of Great Bug Reports
+
+Great Bug Reports tend to have:
+
+- A quick summary and/or background
+- Steps to reproduce
+  - Be specific!
+  - Give sample code if you can. [A stackoverflow question](http://stackoverflow.com/q/12488905/180626) includes sample code that _anyone_ with a base R setup can run to reproduce what I was seeing
+- What you expected would happen
+- What actually happens
+- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
+
+People _love_ thorough bug reports. I'm not even kidding.
+
+## Use a Consistent Coding Style
+
+We are borrowing these from [Facebook's Guidelines](https://github.com/facebook/draft-js/blob/master/CONTRIBUTING.md)
+
+- 2 spaces for indentation rather than tabs
+
```

Comment:
We should use tools to enforce this and not burden contributors

Reply:
Adding "Tools to enforce this to be determined."